app/src/pages/instance/organizationsPage/noAssignedEmptyPage/noAssignedEmptyPage.jsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2019 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useSelector } from 'react-redux';
+import classNames from 'classnames/bind';
+import Parser from 'html-react-parser';
+import { fullNameSelector } from 'controllers/user/selectors';
+import OpenIcon from 'common/img/open-in-rounded-inline.svg';
+import { SAAS } from 'controllers/appInfo/constants';
+import { useIntl } from 'react-intl';
+import RocketIcon from '../img/rocket-inline.svg';
+import FooterIcon from '../img/footer-inline.svg';
+import { messages } from './messages';
+import styles from './noAssignedEmptyPage.scss';
+
+const cx = classNames.bind(styles);
+
+export const NoAssignedEmptyPage = () => {
+  const { formatMessage } = useIntl();
+  const fullName = useSelector(fullNameSelector);
+  const isSaaS = process.env.SAAS !== SAAS;
```

Comment:
The env variables doesn't work in browser.
This should be checked using `instanceTypeSelector`.
cc @maria-hambardzumian

Reply:
Done.